kani-dependencies

```diff
@@ -0,0 +1,2 @@
+CBMC_VERSION="5.61.0"
+CBMC_VIEWER_VERSION="3.5"
```

Comment:
Please keep a comment here pointing to `src/setup.rs`. (And I think there might be a comment there that pointed back to the action? At any rate, that should also have a comment pointing here)

Reply:
Done. I also updated the comment in `src/setup.rs` to point to `kani-dependencies` instead of the action.